generatedocs/data/config/scheduler/examples.md

```diff
@@ -16,3 +18,24 @@
    "instance" : 2
 }
 ```
+
+Simple scheduler set to attempt reconnect on unexpected websocket disconnection 
```

Comment:
Suggestion:
"Simple scheduler set to attempt re-connection in case of an unexpected WebSocket disconnection:"